lib/models/command_info.rb

```diff
@@ -11,7 +11,8 @@ class CommandInfo < T::Struct
       extend JsonHelper
 
       const :desc, String
-      const :run, String
+      const :host_run, T.nilable(String)
```

Comment:
This adds a property but `HCKClient` ignores it. One of the reasons of having a type is to tell the user that there is an extra property and having an extra property in type defeats that purpose.